src/components/dialogs/network-modifications/voltage-level/move-feeder-bays/move-voltage-level-feeder-bays.type.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { ConnectablePositionInfos } from '../../../connectivity/connectivity.type';
+
+export type FeederBaysFormInfos = {
+    equipmentId: string;
+    busbarSectionId: string | null;
+    connectionSide: string | null;
+    connectionName: string | null;
+    connectionDirection: string | null;
+    connectionPosition: string | null;
+    type: string;
+};
+
+export type FeederBayInfos = {
+    busbarSectionId: string;
+    connectionSide: string | null;
+    connectablePositionInfos: ConnectablePositionInfos;
+};
+
+export type FeederBaysInfos = FeederBayInfos & {
```

Comment:
This type is weird. It represents only one FeederBayInfos. So why is it named FeederBay**s**Infos ? To rework ?

Reply:
So the type should be renamed as it is only ONE feeder bay. 
Maybe rename : 
FeederBayInfos -> FeederBayModificationInfos
FeederBaysInfos -> FeederBayInfos

Reply:
Ok my bad. I didn't see it was an array